MAUI/Radio-Button/Grouping.md

```diff
@@ -125,3 +125,47 @@ The [`CheckedChanged`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Butto
 
 ![.NET MAUI RadioGroup horizontal orientation](Images/Grouping/radiogrouporientation.png)
 
+## SelectedValue
```

Comment:
Selected value

Reply:
Updated the Heading from SelectedValue to Selected value